### ISSUE TITLE
Train Cox Proportional Hazards model

### ISSUE THREAD

**Issue by jaren55** [jaren55/BTT-Grid-WiDS-Project#9]
Goal:
Train Cox model for survival analysis and include in ensemble.

Definition of Done:
- Cox model fitted to training data
- Baseline survival computed
- Predictions generated for test set
